pkg/kubewarden/chart/kubewarden/policy-server/index.vue

```diff
@@ -80,6 +122,40 @@ export default {
       }
     },
 
+    updateSecurityContexts({ type, data }) {
+      this.$set(this.chartValues.spec.securityContexts, type, data);
+
+      // check "required" of sysctls
+      // based on https://doc.crds.dev/github.com/kubewarden/kubewarden-controller/policies.kubewarden.io/PolicyServer/v1@v1.9.0#spec-securityContexts
+      const sysctlsCheck = this.chartValues?.spec?.securityContexts?.pod?.sysctls || [];
+      let isSysctlsValid = true;
+
+      if (sysctlsCheck.length) {
+        for (let i = 0; i < sysctlsCheck.length; i++) {
+          if (!sysctlsCheck[i].name || !sysctlsCheck[i].value) {
+            isSysctlsValid = false;
+            break;
+          }
+        }
+      }
+
+      // check "required" of seccompProfile
+      // based on https://doc.crds.dev/github.com/kubewarden/kubewarden-controller/policies.kubewarden.io/PolicyServer/v1@v1.9.0#spec-securityContexts
+      let isSeccompProfileValid = true;
+
+      ['container', 'pod'].forEach((type) => {
+        const seccompProfileContainerCheck = this.chartValues?.spec?.securityContexts?.[type]?.seccompProfile || {};
```

Comment:
Nit: Variable name is a bit misleading here as it could be either a container or pod, perhaps just `seccompProfileCheck` would suffice.

Reply:
Correct. Good catch